### PR TITLE
Enable PWA functionality to examples.

### PR DESCRIPTION
**Pull request by weboko**
Addressed #122 

Added manifest.json to all examples.
Added Waku logo to all examples.

### CODE REVIEW

File: light-chat/index.html [hunk 1]
```diff
@@ -1,50 +1,53 @@
...
-        <script type='module' src="./index.js"></script>
-    </body>
+        <div class="controls">
+          <button id="send">Send</button>
+          <button id="exit">Exit chat</button>
+        </div>
+      </div>
+    </div>
+
+    <script src="//cdn.jsdelivr.net/npm/protobufjs@7.X.X/dist/protobuf.min.js"></script>
```
Comment (danisharora099) [inline light-chat/index.html:50]: unrelated to the scope of this PR; curious why there's a `//` prefixed to the source URL?

**Review by danisharora099 — APPROVED** [review]

File: light-chat/index.html (hunk 1 above)
Comment (weboko) [reply light-chat/index.html:50]: It is called relative protocol and will use protocol of current URL
https://en.wikipedia.org/wiki/URL#prurl

Comment (danisharora099) [reply light-chat/index.html:50]: Interesting!

File: store-js/manifest.json [hunk 2]
```diff
@@ -0,0 +1,20 @@
+{
+  "short_name": "Store js",
```
Comment (fryorcraken) [inline store-js/manifest.json:2]: Is is the name the app will get on the phone? if so, would be good to include `Waku` in it.

**Review by fryorcraken — COMMENTED** [review]
Looks good.

Did not get deployed due to https://github.com/waku-org/js-waku-examples/commit/dd43fac5a2f21ea3348be75a398efbf981b87e6f#r95208997 so could not test.

File: store-js/manifest.json (hunk 2 above)
Comment (weboko) [reply store-js/manifest.json:2]: Addressed it here #182

**Pull request merged**